swagger/pelican-swagger.yaml

```diff
@@ -476,7 +476,19 @@ definitions:
       type:
         type: string
         example: "posix"
-        description: "The storage type of the origin server. Is either \"posix\" or \"s3\""
+        description: "The storage type of the origin server. Values can be posix, s3, and https"
+      s3Region:
+        type: string
+        description: "The server region for AWS S3 storage. Only available when type == s3"
```

Comment:
Let's remove references to AWS, since this is a core part of the protocol (as opposed to the service offered by Amazon)